<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Andromeda </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(20, 20, 20);
color: turquoise; 



}
QLineEdit
{
border-radius: 10px;
border: 1px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31);

}
QLabel
{
background-color: rgb(20, 20, 20);
color: turquoise;
}

QTableView
{
background-color: rgb(20, 20, 20);
color: turquoise;
selection-background-color:  turquoise;
}
QTableView:selected
{
background-color: turquoise;
color: rgb(20, 20, 20);
}
QTableView QHeaderView::section{
    background: rgb(20, 20, 20);
    border: 2px outset rgb(20, 20, 20);
	color: turquoise;
}
QTableView QTableCornerButton::section {
    background: rgb(20, 20, 20);
    border: 2px outset rgb(20, 20, 20);
	color:turquoise;
}
QTableView QScrollBar{

background-color: turquoise;
color: rgb(20, 20, 20);
}
QMessageBox
{
background-color: rgb(20, 20, 20);
color: turquoise;
}

QPushButton{
	border-radius: 7px;
    color: black; 
    background: turquoise;
}
QPushButton:hover{
	border-radius: 7px;
	color: turquoise; 
    background: black;
	
}
QToolTip{
border-radius: 10px;
border: 1px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31);
}
QGroupBox{
border-radius: 10px;
border: 1px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31);
}


QRadioButton{
color: turquoise;
background-color: rgb(31, 31, 31);
}

indicator:checked:pressed{
color: turquoise;
background-color: rgb(31, 31, 31);
selection-background-color:  turquoise;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>434</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="15" column="3">
      <widget class="QLineEdit" name="lineEdit_19">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Коэффициент полезного действия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>КПД , %</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Время перелёта, сут</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>35786</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="text">
        <string>40</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>60</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Масса рабочего тела, кг</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Удельная масса двигателя, кг/Н</string>
       </property>
      </widget>
     </item>
     <item row="17" column="3">
      <widget class="QLineEdit" name="lineEdit_21">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Скорость истечения, м/c</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Относительная масса конструкции</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>400</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="text">
        <string>15000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="3">
      <widget class="QLineEdit" name="lineEdit_22">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Высота начальной орбиты, км</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Высота конечной орбиты, км</string>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Масса полезной нагрузки,кг</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="text">
        <string>0.01</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Характерическая скорость, м/с</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>51</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Мощность энергоустановки, КВт</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>10000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="3">
      <widget class="QLineEdit" name="lineEdit_20">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="label_19">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Масса системы подачи и хранения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Масса СПХ, кг</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_22">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Удельная масса энергоустановки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Удельная масса ЭУ, кг/Вт</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Угол наклона начальной орбиты, гр</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Стартовая масса, кг</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Тяга двигательной установки, кг</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Угол наклона конечной орбиты, гр</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Масса двигательной установки, кг</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="text">
        <string>0.07</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="text">
        <string>0.1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_28">
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31); </string>
       </property>
       <property name="text">
        <string>Орбита</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31); </string>
       </property>
       <property name="text">
        <string>Результаты вычислении:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_29">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Параметры двигательной и энергетической установок&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31); </string>
       </property>
       <property name="text">
        <string>ДУ и ЭУ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Скорость на круговой орбите, км/c</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>112</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_23">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Удельная масса системы подачи и хранения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Удельная масса СПХ, кг/Н</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Масса энергоустановки, кг</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31); </string>
       </property>
       <property name="text">
        <string>Начальные параметры:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Масса конструкции, кг</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>681</width>
      <height>181</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Кликнув выбрать номер двигателя в колонке &amp;quot;Код&amp;quot;&lt;/p&gt;&lt;p&gt;Нажать Enter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(images/logo_BG.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string> Режим расчёта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Гипотетический</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбор двигателя</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Оптимизация по двигателю</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(31, 31, 31); */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>730</y>
      <width>148</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 31, 31);</string>
    </property>
    <property name="text">
     <string>Выбран двигатель</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>730</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>740</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Рассчёт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>740</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зависимость наклонения орбиты (град) от времени (сут)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>661</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(20, 20, 20);
</string>
    </property>
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>661</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid turquoise;
color: turquoise;
background-color: rgb(20, 20, 20);
</string>
    </property>
    <widget class="QCustomPlot" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>680</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зависимость радиуса орбиты (км) от времени (сут)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_21</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>tableView</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_27</zorder>
   <zorder>label_26</zorder>
   <zorder>lineEdit_23</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_31</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>label_32</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
